docs/materials/day1/part1-ex5-request.md

```diff
@@ -57,7 +57,7 @@ The Resident Set Size (`RSS`) column, highlighted above, gives a rough indicatio
 Using `top`:
 
 ``` console
-username@learn $ top -u %RED%username%ENDCOLOR%
+username@learn $ top -u %RED%<username>%ENDCOLOR%
```

Comment:
@lmichael107 can you make this `<USERNAME>` instead?

Reply:
done.